index.js

```diff
@@ -226,4 +227,13 @@ const preprocessCheck = (spec, value) => {
 	return (spec.preprocess) ? spec.preprocess(value) : value
 }
 
+const startCheck = (specs) => {
+	let nextStart = 1
+	specs.map.forEach(col => {
```

Comment:
You should use a map function here instead of forEach.

Reply:
something like [this](https://github.com/SteveyPugs/fixy/pull/41/commits/2fc625f1c69ab2dd12d329f45e00c6f04925d437)?

Reply:
from my undestanding, with using map instead of forEach I'm creating a new specs.map array instead of modifying the existing one, but the original specs is modified anyway (now points to a different map array)

Reply:
Yes. Correct. ForEach is more for looping and review. Map is for modifications and also is a bit more speeder.